### PR TITLE
Make hulks speak like an ork

### PR DESCRIPTION
**Pull request by optimumtact**
## About The Pull Request
Makes hulks speak like an ork

## Why It's Good For The Game
CRUMP HUMIES MORE DAKKA
## Changelog
:cl:oranges
add: Hulks now talk like an ork
/:cl:

Suggest more word replacements

### CODE REVIEW

**Review by Kylerace — APPROVED** [review]

**Comment by 567Turtle** [comment]
I like it :)

**Comment by 567Turtle** [comment]
Actually I just got an idea, you can keep this tied to the hulk so the hulk code doesn't need to change, but you should also make a new mutation kind of like the Chav mutator which makes it so you can talk like this without being big bad hulk

**Comment by DAKKA-WAAAGH** [comment]
### **BASED**

File: strings/ork_replacement.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	  "nuke": "bomb",
+	  "ai": "talky machine",
+	  "artificial intelligence": "talky machine",
+	  "shuttle": "winged trukk",
+	  "miner": "rockcrusher",
+	  "mining": "rock crushing",
+	  "maintenance": "tunnel",
+	  "maint":"tunnel",
+	  "singularity": "space hole",
+	  "supermatter": "angry rock"
```
Comment (Kylerace) [inline strings/ork_replacement.json:53]:
```suggestion
	  "supermatter": "angry rock",
	  "war": "WAAAGH!",
	  "hos": "nob"
```

**Review by Kylerace — COMMENTED** [review]
🤓

**Comment by Colovorat** [comment]
give it a chance for any word to be replaced with WAAAGH!!, something like 1 in 1000

**Comment by Shadowflame909** [comment]
should have antag phrases dumbed down too. Like traitor into meanie and changeling into changey thing

**Comment by YakumoChen** [comment]
Have any diety-type name replaced with a random choice between Gork and Mork

**Comment by YakumoChen** [comment]
The = da
One = wun
Fast / faster = fasta

**Comment by theta060** [comment]
best PR since the TTS one

**Comment by optimumtact** [comment]
WIP until I get some more replacies

**Comment by Sealed101** [comment]
exosuit, mech, mecha - killa kan
rocket - rokkit
flamethrower - burna
doctor, MD, doc, medic - dok
CMO - mad dok
ninja - kommando / purple git
nukies/nuke ops - flash gitz
CE - big mek

could consider replacing any race-specific callouts with ### gitz, i. e. scalie gitz, burna gitz etc

some authentic ork speech patterns like foregoing H in the beginning of words or G in -ing or R in -er exist but that could get bothersome

**Comment by Kitfox88** [comment]
Leave hulks as they are but introduce Ork Freebootas as a pirate raid option that use this

**Comment by optimumtact** [comment]
> Leave hulks as they are but introduce Ork Freebootas as a pirate raid option that use this
no

**Comment by Farquaar** [comment]
monkey = gobbo

**Comment by theta060** [comment]
id imagine one of the warhammer servers to already have this stuff couldnt you just copy that?

**Comment by optimumtact** [comment]
ok why don't you do that then instead of pinging everyone

**Comment by SuperNovaa41** [comment]
Reopen this when you plan on finishing it

**Pull request closed without merging**